src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/PrisonController.kt

```diff
@@ -7,15 +7,13 @@ import org.springframework.http.HttpStatus
 import org.springframework.http.MediaType
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
-import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.whereabouts.services.LocationService
 import uk.gov.justice.digital.hmpps.whereabouts.services.locationfinder.LocationIdAndDescription
 
 @Api(tags = ["prison"])
 @RestController
```

Comment:
Can we rename this from PrisonController to Videolink booking rooms - it's fine for it to be its own resource.

You'd expect a PrisonController to return Prisons

Reply:
renamed as follows
PrisonController -> VideoLinkBookingRoomsController
PrisonControllerTest -> VideoLinkBookingRoomsControllerTest
PrisonIntegrationTest -> VideoLinkBookingRoomsIntegrationTest